src/api.js

```diff
@@ -432,14 +452,28 @@ class API {
       if (this._widgetOpened) {
         this._widgetOpened = false;
 
-        if (this[callback]) this[callback](payload);
```

Comment:
Same here, I didn't think we'd need to change any of this. It could be the same as before and we'd only need to make the change from Vezgo Connect to keep this simple.